File: QE_FST_Selenium_java/Activity19_Alerts2.java
package demos;

import org.openqa.selenium.Alert;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.By;
public class Activity19_Alerts2 {
    public static void main(String[] args) {
        WebDriver driver=new FirefoxDriver();
        driver.get("https://training-support.net/webelements/alerts");
        System.out.println(driver.getTitle());
        WebElement confirm=driver.findElement(By.id("confirmation"));
        confirm.click();
        Alert alert=driver.switchTo().alert();
        System.out.println(alert.getText());
        alert.accept();
        System.out.println(driver.findElement(By.id("result")).getText());
        confirm.click();
        driver.switchTo().alert();
        alert.dismiss();
        System.out.println(driver.findElement(By.id("result")).getText());
        driver.quit();
    }
}

File: QE_FST_Selenium_python/Activity19_Alerts2.py
from selenium import webdriver
import time
from selenium.webdriver.common.by import By
with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/alerts")
    print(driver.title)
    confirm=driver.find_element(By.ID,"confirmation")
    confirm.click()
    alert = driver.switch_to.alert
    time.sleep(1)
    print(alert.text)
    alert.accept()
    res=driver.find_element(By.ID,"result")
    print(res.text)
    confirm.click()
    driver._switch_to.alert
    alert.dismiss()
    print(res.text)


    
